src/tokens.c

```diff
@@ -37,6 +38,23 @@ const tokenDefinition_t const TOKENS_EXTRA[NUM_TOKENS_EXTRA] = {
       0x53, 0x3c, 0xd6, 0xa2, 0x1e, 0x3e, 0x59, 0xbe, 0x32, 0x4a},
      "tBTC ",
      18},
+     // Goerli DeversiFi tokens
+    {{0xd9, 0x97, 0xa8, 0x63, 0x46, 0xe7, 0x65, 0x18, 0xe6, 0x92,
+      0x25, 0x56, 0xf3, 0x4d, 0x76, 0x61, 0x30, 0xc0, 0xbb, 0xfd},
+     "tUSDT ",
+     6},
+    {{0xc1, 0xd5, 0x79, 0xeb, 0xff, 0x7c, 0x0f, 0x6c, 0xfd, 0x9a,
+      0xd5, 0xfb, 0x26, 0x7f, 0xec, 0x73, 0xbe, 0x70, 0xc8, 0xf7},
+     "tBTC ",
+     18},
+    {{0xa8, 0xf3, 0x14, 0x4f, 0xea, 0x2c, 0x37, 0x5a, 0xd0, 0x58,
+      0xec, 0x12, 0x09, 0x9a, 0x5a, 0x21, 0xa2, 0x6f, 0xe9, 0x96},
+     "tDVF ",
+     18},
+    {{0x4c, 0xda, 0xbe, 0xc1, 0x2a, 0x39, 0x7f, 0xb6, 0xef, 0xaf,
+      0x46, 0x13, 0xd5, 0xdf, 0xd7, 0x9b, 0x30, 0x9a, 0xe9, 0xfa},
+     "tXDVF ",
+     18},
```

Comment:
As far as I know this token.c file will not be used. 
Instead we should add your token to the list of ERC20 supported tokens. (cc @TamtamHero for confirmation)

Can you please confirm these same contract don't already exist on Etheruem mainnet ?

Reply:
Adding those tokens here + adding `HAVE_TOKENS_EXTRA_LIST` for `goerli` in the Makefile did work for our cases (the goal is for us to be able to call `provideERC20TokenInformation` with `00{tokenAddress}...{chainId}`.
Before adding those tokens in that file, the above won't work. After adding them I've got the behavior I expected.
The tokens in the list have been freshly minted on Goerli with fresh smart-contract addresses. It doesn't collude with anything on mainnet.

Reply:
@FabriceDautriat  It works because Goerli compiles as a standalone Eth app, unlike clones (cf `DEFINES_LIB=` in the Makefile, same for ropsten)